app/javascript/App.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import fetchData from './redux/api';
+
+function App() {
+  const dispatch = useDispatch();
+  const { greeting } = useSelector((state) => state.greeting);
+
+  React.useEffect(() => {
+    dispatch(fetchData());
+  }, []);
+
+  return <h1>{greeting}</h1>;
+}
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements:

![image](https://github.com/Elerqsousy/react-on-rails-one-app/assets/77069011/40ca2d17-a000-4493-a6ce-69c3c28427bc)

Please consider creating a separate component `Greeting` that is responsible for displaying the greeting coming from the backend. Also, consider using `react-router-dom` to set up a route for this component.